Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -7,8 +7,13 @@ import XCTest
 class InAppMessageViewTest: UnitTest {
     private let queueMock = MessageQueueManagerMock()
     private let engineWebMock = EngineWebInstanceMock()
-    private let inlineMessageDelegateMock = InlineMessageDelegateMock()
+    private let inlineMessageDelegateMock = InAppMessageViewActionDelegateMock()
     private var engineProvider: EngineWebProviderStub2!
+    private let eventListenerMock = InAppEventListenerMock()
+    private var messagingInApp: MessagingInAppImplementation {
+        // get MessagingInAppImplementation instance so we can call its methods directly
+        (MessagingInApp.shared.implementation as! MessagingInAppImplementation) // swiftlint:disable:this force_cast
+    }
```

Comment:
I think it would be helpful if we moved this into the parent `UnitTest` class so it can be re-used by other tests in the suite.